templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>Delete car</h1>
+  <p>Are you sure, you want to delete car: {{ car }}? Deleted data cannot be returned</p>
+  <form action="" method="post">
+    {% csrf_token %}
+    {{ form|crispy }}
+    <input type="submit" value="Yes, delete" class="btn btn-danger"/>

```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on -- checklist

Reply:
in manufacturer confirm delete too)